ush/parsing_namelists_FV3.sh

```diff
@@ -42,6 +42,18 @@ local SDAY=${current_cycle:6:2}
 local CHOUR=${current_cycle:8:2}
 local MOM6_OUTPUT_DIR="./MOM6_OUTPUT"
 
+if (( OFFSET_START_HOUR > 0 )); then
```

Comment:
Since this is only used for replay, should we change the conditional to `REPLAY_ICS`?

Reply:
Conditional has been changed to `REPLAY_ICS`.